src/store/reducers/devicesReducer.js

```diff
@@ -132,7 +143,8 @@ const updatePropertiesReducer = (state, { payload }) => {
 
 /* Action types that cause a pending flag */
 const fetchableTypes = [
-  epics.actionTypes.fetchDevices
+  epics.actionTypes.fetchDevices,
+  epics.actionTypes.fetchDevicesByCondition
```

Comment:
Are multiple epics really needed?  One fetches all (so conditions are blank) and one fetches based on input conditions?

Reply:
One fetches based on existing condition in redux. Another fetched based on condition sent in payload

Reply:
done